src/Console/Deployer.php

```diff
@@ -59,7 +59,7 @@ function ($value, $key) {
             array_keys($parameters)
         ));
 
-        $process = new Process('serverless invoke local ' . $p, '.bref/output');
+        $process = new Process(['serverless invoke local ' . $p, '.bref/output']);
```

Comment:
I think the way the new syntax should be with Symfony\Process is something like this:

```php
new Process(['serverless', 'invoke', 'local', $p], '.bref/output'])
```

And I think `$p` should be split too for each argument. No need to `implode()` and escape ourselves the arguments it seems: https://symfony.com/doc/current/components/process.html#using-features-from-the-os-shell